src/tests/unit/containers/orders/order/empty_order/tests/empty-order.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { ThemeProvider } from '@material-ui/core';
+import EmptyOrder from '../../../../../../../containers/orders/order/empty-order/index';
+import { theme } from '../../../../../../../components/app/app-theme/app.theme';
+
+const themeValue = theme('light');
+
+describe('Empty order component tests', () => {
+  const buttonTitle = 'до каталогу';
+  const emptyTitle = 'Ваш кошик порожній';
+
+  it('should render title', () => {
+    const { container } = render(
```

Comment:
you don't need to write a render in every test case
use beforeEach

Reply:
Fixed, thanks